app/views/planning_applications/_form.html.erb

```diff
@@ -34,7 +34,9 @@
         <% if @planning_application.open_description_change_requests.any? %>
           <%= link_to "View requested change", planning_application_description_change_validation_request_path(@planning_application, @planning_application.open_description_change_requests.last.id), class: "govuk-link" %>
         <% else %>
-          <%= link_to "Propose a change to the description", new_planning_application_description_change_validation_request_path(@planning_application), class: "govuk-link" %>
+          <% unless @planning_application.closed_or_cancelled? %>
+            <%= link_to "Propose a change to the description", new_planning_application_description_change_validation_request_path(@planning_application), class: "govuk-link" %>
+          <% end %>
         <% end %>
```

Comment:
Rather than nesting an `unless` block in an `else` block, could you just have `elsif !@planning_application.closed_or_cancelled?`